hadoop-hdds/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockInputStreamProxy.java

```diff
@@ -117,6 +117,8 @@ private synchronized void setReaderType() {
     int expected = expectedDataLocations(repConfig, getLength());
     int available = availableDataLocations(blockInfo.getPipeline(), expected);
     reconstructionReader = available < expected;
+    LOG.info("Available data locations: {}, Expected data locations: {}. Based on this online reconstruction is: {}",
+        available, expected, reconstructionReader ? "triggered." : "not triggered.");
```

Comment:
I don't think the message is needed for normal ("not triggered") case.  Also, please reduce verbosity, something like:

```
Data locations available: {} < expected: {}, using reconstruction read
```